### PR TITLE
Littlefoot/statics/fix bug get test fixture attribute

### PR DESCRIPTION
**Pull request by LiyueWang**

### CODE REVIEW

**Comment by LiyueWang** [comment]
run it locally and successful

File: NUnitTestCategoryAssertor/Scanner/TestAssemblyScanner.cs [hunk 1]
```diff
@@ -150,5 +154,33 @@ IEnumerable<string> testFixtureCategories
...
+			if( !nunitAssemblies.Any() ) {
+				return false;
+			}
+
+			foreach( AssemblyName nunitAssembly in nunitAssemblies ) {
+
+				Version version = nunitAssembly.Version;
+				if( !version.Equals( expectedVersion ) ) {
+
+					string msg = $"Expected { expectedAssemblyName } version { expectedVersion }, but found { version }";
```
Comment (saquibmian) [inline NUnitTestCategoryAssertor/Scanner/TestAssemblyScanner.cs]: nit: would be nice to log which assembly the unexpecter version was found in

File: NUnitTestCategoryAssertor/packages.config [hunk 2]
```diff
@@ -2,5 +2,5 @@
 <packages>
   <package id="NDesk.Options" version="0.2.1" targetFramework="net45" developmentDependency="true" />
   <package id="NDesk.Options.TypedOptionSet" version="1.0.1" targetFramework="net40" developmentDependency="true" />
```
Comment (saquibmian) [inline NUnitTestCategoryAssertor/packages.config]: Are these supposed to be `developmentDependency`? They are required for the tool to work, no? @JeffAshton

Comment (JeffAshton) [inline NUnitTestCategoryAssertor/packages.config]: Yes.  This is packaged as a tool, so the dependencies shouldn't be visible.  They're packaged in

Comment (saquibmian) [inline NUnitTestCategoryAssertor/packages.config]: Ah the tool bundles it. Nevermind.

**Pull request merged**